### DISCUSSION TITLE
Elasticsearch "FAIL" and other setup questions

### DISCUSSION THREAD

**Discussion by gbossley** [SEPIA-Framework/sepia-docs#120] [Q&A]
I've tried to follow the Quickstart instructions, but keep getting a fail on running option 4, "Start Elasticsearch"

How do i 'check console for errors'? I have no idea where / how to start with this tip.

How do I confirm I have correctly setup up Java and OpenJDK 11?

I'm using Java version 8 update 301 (build 1.8.0_301-b09).

**Comment by fquirin** [comment]
Hi Gordon,

I think you are on Windows if I remember right?

Can you first first check if Java is properly installed by typing `java -version` in your terminal? Both Java 8 and 11 are fine.

To test Elasticsearch itself please open your SEPIA folder and go to `[SEPIA]\elasticsearch\bin`. There you can use `elasticsearch.bat` to run the DB alone and check the terminal for errors. On Java 11 there will be a deprecation warning at the beginning but you can ignore it ;-).

**Reply by gbossley** [reply]
Hi Florian,
Per your suggestions, my java version is "1.8.0_301". I understand that means Java 8.
I have my Sepia folder on my secondary hard drive. Is this going to be the cause of problems (I think 'perhaps')?
On running elasticsearch.bat:
'Files' is not recognized as an internal or external command,
operable program or batch file.
Could not find any executable java binary. Please install java in your PATH or set JAVA_HOME
Press any key to continue . . .

Thanks in advance.
Gordon.

**Reply by fquirin** [reply]
> I have my Sepia folder on my secondary hard drive. Is this going to be the cause of problems (I think 'perhaps')?

For Windows this shouldn't be any problem.

> 'Files' is not recognized as an internal or external command

This is strange, I have no idea where 'Files' is supposed to be called :thinking: - What version of Windows do you have? Windows 10 64bit?

> Could not find any executable java binary. Please install java in your PATH or set JAVA_HOME

For some reason even though you can use 'java -version' it seems your Java isn't properly installed or at least not registered in your PATH. Can you try the following please:
- Go to `[SEPIA]/java` and look for `download_java_11_win64.bat` (assuming you have Windows 64bit)
- Double-click the batch file and wait until Java 11 is downloaded into your local SEPIA folder (~300MB)
- Go back and double-click the `setup.bat` to start the SEPIA setup
- Check to first entry, it should show something like "found local Java version ..."
- If the local Java version was found try 4) to start Elasticsearch again

**Reply by gbossley** [reply]
"SUCCESS"!!!

Thanks Florian!

I think I am very much out of my depth playing with your beast...
Trying to create an account and log in...

 "result": "fail",
  "error": "400 or 500 bad request or communication error",
  "code": "0"

**Reply by fquirin** [reply]
> "SUCCESS"!!!

Great! :-)

> I think I am very much out of my depth playing with your beast...

Any feedback is very welcome since in the end the goal is that everybody feels comfortable playing around with SEPIA. I know its still a bit rough around the edges :sweat_smile: 

> Trying to create an account and log in...

Is this the message you get inside the SEPIA Control-HUB during login? You might get this when the username is wrong. Try 'uid1003' and the password you've assigned during setup.
